### PR TITLE
Adding ngnix stats

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

**Review by sandersms — APPROVED** [review]
Looks good

File: config/telegraf-spdk.conf [hunk 1]
```diff
@@ -17,6 +17,9 @@
   tag_keys = ["name"]
   json_query = "result.bdevs"
 
+[[inputs.nginx]]
```
Comment (seroyer) [inline config/telegraf-spdk.conf:20]: I'm not sure why nginx stats are reported via a conf file with spdk in the name.  Are they related in some way?

File: config/nginx.conf [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    #error_page  404              /404.html;
+
+    # redirect server error pages to the static page /50x.html
+    #
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
+    # proxy the PHP scripts to Apache listening on 127.0.0.1:80
```
Comment (seroyer) [inline config/nginx.conf:28]: This could probably be cleaned up.

File: config/telegraf-spdk.conf (hunk 1 above)
Comment (glimchb) [reply config/telegraf-spdk.conf:20]: I can rename the file and remove spdk from the name, thanks @seroyer good comment

File: config/nginx.conf (hunk 2 above)
Comment (glimchb) [reply config/nginx.conf:28]: I thought about that as well... 
I created this file using 
```
docker run --rm -it docker.io/library/nginx:1.23.3-alpine cat /etc/nginx/conf.d/default.conf
```
so was on the fence leaving it like this or clean-up... thanks for the input

**Pull request merged**